elfi/methods/bo/acquisition.py

```diff
@@ -411,6 +413,117 @@ def evaluate_gradient(self, theta_new, t=None):
         return gradient
 
 
+class RandMaxVar(MaxVar):
+    r"""The randomised maximum variance acquisition method.
+
+    The next evaluation point is sampled from the density corresponding to the
+    variance of the unnormalised approximate posterior (The MaxVar acquisition function).
+
+    \theta_{t+1} ~ Var(p(\theta) * p_a(\theta)),
+
+    where the likelihood is defined using the CDF of normal distribution, \Phi, as:
+
+    p_t(X_t) =
+        (\Phi((\epsilon - \mu_{1:t}(\theta)) / \sqrt(\sigma_{1:t}^2(\theta) + \sigma_n^2))) * C,
+
+    where \epsilon is the discrepancy threshold, \mu_{1:t} and \sigma_{1:t} are
+    determined by the Gaussian process, \sigma_n is the noise, and C is the normalisation constant.
+
+
+    References
+    ----------
+    [1] arXiv:1704.00520 (Järvenpää et al., 2017)
+
+    """
+
+    def __init__(self, quantile_eps=.01, n_mh_samples=10000, cov_mh=np.ones(1),
```

Comment:
I think cov_mh might be a bit confusing name as only the variances of the covariance matrix of the Gaussian proposal are given as input (Am I correct with this?).
Also, does this work on all dimensions of parameter space?
Would it be possible to set the default variances according to the bounds of the parameter space (instead of just 1)?

Reply:
@vuolleko, is it true that `sigma_proposals` of `metropolis` correspond to the variances of the covariance matrix of the Gaussian proposal?

Reply:
No, sigma is the standard deviation.

Reply:
Changed the naming of `cov_mh` to `sigma_proposals_metropolis`.

> Would it be possible to set the default variances according to the bounds of the parameter space (instead of just 1)?
Could you expand on how to find the default variances according to the bounds?

Reply:
Maybe one could set the standard deviation to be e.g. the length of the interval/10. I think it could be a good default. Having 1 can be problematic, e.g. if the parameter space is e.g. [0,1000] and the posterior support is similar

Reply:
The sigmas are now set to the interval/10 as suggested.